### PR TITLE
removed references to greymatter-minikube.yaml

### PR DESCRIPTION
**Pull request by dgoldstein1**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

- additional update documentation relating to `greymatter-minikube.yaml` after deleting `greymatter-minikube.yaml` file in https://github.com/DecipherNow/helm-charts/pull/341

2. What **changes to custom.yaml** are required?

none

3. Have you documented any additional setup steps or configurations options?

no documentation required

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

- no UI changes

closes https://github.com/DecipherNow/helm-charts/issues/347

### CODE REVIEW

File: docs/Deploy with Minikube.md [hunk 1]
```diff
@@ -301,7 +300,7 @@ Then you can run the following commands to update the local charts and then inst
 
 ```sh
 helm dep up greymatter
-helm install greymatter -f custom-greymatter.yaml -f custom-greymatter-secrets.yaml -f custom-reymatter-minikube.yaml --name gm
+helm install greymatter -f custom-greymatter.yaml -f custom-greymatter-secrets.yaml --name gm
```
Comment (chrisbsmith) [inline docs/Deploy with Minikube.md]: This need to also reference `-f credentials.yaml`

File: docs/Deploy with Minikube.md [hunk 2]
```diff
@@ -315,7 +314,7 @@ We also have the option to specify:
 - "--replace" will replace an existing deployment
 - "--dry-run" will print all Kubernetes configs to stdout
 
-We can run `helm ls` to see all our current deployments and `helm delete --purge $DEPLOYMENT` to delete deployments. If you need to make changes, you can run `helm upgrade gm deciperhnow/greymatter -f custom-greymatter.yaml -f custom-greymatter-secrets.yaml -f custom-greymatter-minikube.yaml` to update your release in place.
+We can run `helm ls` to see all our current deployments and `helm delete --purge $DEPLOYMENT` to delete deployments. If you need to make changes, you can run `helm upgrade gm deciperhnow/greymatter -f custom-greymatter.yaml -f custom-greymatter-secrets.yaml` to update your release in place.
```
Comment (chrisbsmith) [inline docs/Deploy with Minikube.md]: Another place that needs to reference `credentials.yaml`

**Comment by whenhellfreezes** [comment]
The changes to the minikube make targets are covered in my s3config-make PR.

File: docs/Deploy with Minikube.md (hunk 1 above)
Comment (dgoldstein1) [reply docs/Deploy with Minikube.md]: I'm confused. There isn't any `credentials.yaml`

**Comment by tilleryd** [comment]
@dgoldstein1 In https://github.com/DecipherNow/helm-charts/pull/341 you also added.

```
```yaml
  environment: minikube
```
```

That needs to be:

```
```yaml
  environment: kubernetes
```
```

File: docs/Deploy with Minikube.md (hunk 1 above)
Comment (tilleryd) [reply docs/Deploy with Minikube.md]: @chrisbsmith credentials.yaml is only applicable when running the Makefile

Comment (chrisbsmith) [reply docs/Deploy with Minikube.md]: ah, thanks for the clarification

File: docs/Deploy with Minikube.md [hunk 3]
```diff
@@ -259,7 +258,7 @@ helm repo add decipher https://nexus.production.deciphernow.com/repository/helm-
 helm repo update
 ```
 
-Now let's tell our helm chart to use `minikube` as its environment. In `greymatter.yaml` change 
+We also need to tell our helm chart to use `minikube` as its environment. In `greymatter-custom.yaml` change
```
Comment (tilleryd) [inline docs/Deploy with Minikube.md]: this should be `custom-greymatter.yaml`

File: docs/Deploy with Minikube.md [hunk 4]
```diff
@@ -276,7 +275,7 @@ Once the repository has successfully been added to your `helm` CLI, and our envi
 **Note: Before installing Helm charts it's always prudent to do a dry-run first to ensure your custom YAML is correct. You can do this by adding the `--dry-run` flag to the below `helm install` command. If you receive no errors then you can confidently drop the `--dry-run` flag.**
 
 ```sh
-helm install greymatter -f greymatter.yaml -f greymatter-secrets.yaml  --name gm-deploy
+helm install decipher/greymatter custom-greymatter-secrets.yaml custom-greymatter.yaml --name gm-deploy
```
Comment (tilleryd) [inline docs/Deploy with Minikube.md]: the `-f` flags are required

Comment (tilleryd) [reply docs/Deploy with Minikube.md]: and since you're making changes to this line please match `--name gm` with the local install command

**Comment by dgoldstein1** [comment]
@tilleryd thanks for your comments, I've made changes to the docs.

At some point we should have `environment` be more than just `openshift` or `kubernetes`, but something like `gke`, `baremetal`. Otherwise, we should add a new field `provider`

**Comment by tilleryd** [comment]
@dgoldstein1 thanks, can you write up an issue detailing your suggestion for using `provider`?

**Comment by dgoldstein1** [comment]
https://github.com/DecipherNow/helm-charts/issues/351

**Pull request merged**